### ISSUE TITLE
How to disable animation between tabs?

### ISSUE THREAD

**Issue by taschik** [react-navigation/react-navigation#7274]
### Current behaviour

![ezgif com-video-to-gif](https://user-images.githubusercontent.com/351623/68711476-8a0e4380-054e-11ea-8887-2bbf7ccf2f30.gif)

Switching between tabs in a BottomTabNavigator animates the nested screen from the bottom to the top.

### Expected behaviour

No animation happen between switching tabs. The screen in the tab should toggle in right away.

### Code sample

```
NotificationsNavigator: {
      screen: NotificationsNavigator,
      navigationOptions: ({ navigation }) => ({
        tabBarIcon: () => {
          const isFocused = navigation.isFocused('NotificationsNavigator')
          return <AppNavigatorHOC isFocused={isFocused} navigator={TAB_NAVIGATOR.NOTIFICATION} />
        },
        tabBarLabel: () => {
          const isFocused = navigation.isFocused('NotificationsNavigator')
          return (
            <Text
              style={[styles.labelStyle, isFocused ? styles.labelActive : styles.labelInactive]}
            >
              {I18n.t('tab_bar_notifications')}
            </Text>
          )
        },
      }),
    },
    ProfileNavigator: {
      screen: ProfileNavigator,
      navigationOptions: ({ navigation }) => ({
        tabBarIcon: () => {
          const isFocused = navigation.isFocused('ProfileNavigator')
          return <AppNavigatorHOC isFocused={isFocused} navigator={TAB_NAVIGATOR.PROFILE} />
        },
        tabBarLabel: () => {
          const isFocused = navigation.isFocused('ProfileNavigator')
          return (
            <Text
              style={[styles.labelStyle, isFocused ? styles.labelActive : styles.labelInactive]}
            >
              {I18n.t('tab_bar_profile')}
            </Text>
          )
        },
      }),
    },
  },
  {
    initialRouteName: 'DiscoveryNavigator',
    headerMode: 'screen',
    animationEnabled: false,
    lazy: true,
    transitionConfig: () => ({ screenInterpolator: () => null }),
    defaultNavigationOptions: ({ navigation }) => defaultNavOptions(navigation),
    tabBarOptions: {
      showIcon: true,
      showLabel: true,
      labelStyle: {
        fontFamily: $regularFontFamily,
        fontSize: 9,
        textAlign: 'center',
        color: 'white',
      },
      inactiveTintColor: $placeholderGrey,
      activeTintColor: 'white',
      style: {
        backgroundColor: $tempoBlack,
        borderTopWidth: 0,
      },
    },
```

**NotificationNavigator**
this is a regular StackNavigator.
```
const NotificationsNavigator = createStackNavigator(
  {
    NotificationsList: {
      screen: NotificationsList,
    },
  },
  {
    initialRouteName: 'NotificationsList',
    defaultNavigationOptions: {
      ...defaultHeaderOptions,
      title: I18n.t('title_notifications').toUpperCase(),
      headerBackTitle: null,
    },
  }
)
```

### What have you tried

Knowing that there are no animations in the tabNavigator, I am kind of clueless but tried adding this to the options ( see code example above)

` transitionConfig: () => ({ screenInterpolator: () => null }),`

### Your Environment
```
"react-navigation": "^4.0.10",
"react-navigation-stack": "^1.10.3",
"react-navigation-tabs": "^2.5.6"
```
| software              | version
| --------------------- | -------
| ios or android        | both
| react-native          | 0.60.4
| node                  | v10.16.3 
| npm or yarn           | npm

**Comment by chuga2310** [comment]
I have same issue with version 4

**Comment by AricWu** [comment]
Hey, I tried disabling the transition animation between all pages, but failed to solve this animation similar to fade in and fade out. No effect. Finally, I found that this is not an animation but a parameter derived from the performance optimization of react navigation for switching pages.
On `node_modules/@react-navigation/bottom-tabs/src/ResourceSavingScene.tsx`,line 36
(My @react-navigation/bottom-tabs version is 5.2.5)
```js
 removeClippedSubviews={
          Platform.OS === 'ios' ? !isVisible : false
        }
```
Change it to 
```
removeClippedSubviews={
          Platform.OS === 'ios' ? !isVisible : true
        }
```
It should be fixed.
